Makefile

```diff
@@ -1,20 +1,22 @@
-BUILD_DIR = build
-DESCRIPTION = CS50 Library for Python
-MAINTAINER = CS50 <sysadmins@cs50.harvard.edu>
-NAME = python-cs50
-OLD_NAME = lib50-python
-VERSION = 1.2.4
+BUILD_DIR := build
+LIB_DIR := $(BUILD_DIR)/usr/lib
+PYTHON2_DIR := $(LIB_DIR)/python2.7/dist-packages/cs50
```

Comment:
Not sure yet! But even `sdist` (just emailed about) seems to assume this path, by default.